tests/test_TFNetworkLayer.py

```diff
@@ -6806,6 +6806,39 @@ def test_ConvLayer_empty_out():
     assert out.shape == (1, 0, 7)
 
 
+def test_ConvLayer_stride_dilation():
+  import numpy as np
+  import torch
```

Comment:
Do not use PyTorch in the test.

Reply:
Whats best then: Should I use a different library than PyTorch/TensorFlow or should I simply hardcode the expected result of the convolution?

Reply:
You can implement it manually using NumPy. Or you can hardcode the expected result. Whatever is easier. The numbers should be as minimal as possible anyway.

Reply:
Or you can just ignore this also. It's not really so important.